nft-simple/src/nft_core.rs

```diff
@@ -57,9 +56,11 @@ trait NonFungibleTokenApprovalsReceiver {
         owner_id: AccountId,
         approval_id: U64,
         msg: String,
-    ) -> Promise;
+    );
 }
 
+// TODO: create nft_on_revoke
+
```

Comment:
Why?

Reply:
I think it's awkward to have `nft_on_approve` but not `nft_on_revoke`. It felt strange as I built the Exquisite Corpse